apps/forum/vars/vault/main.yml.j2

```diff
@@ -6,10 +6,10 @@
 {% set mongodb_credentials = databases.mongodb | json_query("[?release=='" ~ forum_mongodb_version ~ "'].databases | [0][?application=='forum'].{user: user, password: password, name: name} | [0]") %}
 MONGODB_ADMIN_PASSWORD: password
 MONGODB_USER: {{ mongodb_credentials.user }}
-MONGODB_PASSWORD: {{ mongodb_credentials. password }}
+MONGODB_PASSWORD: {{ mongodb_credentials.password }}
 MONGODB_DATABASE: {{ mongodb_credentials.name }}
 
 # Forum
 # Nota bene: the forum API_KEY should match the following edxapp lms settings:
 # COMMENTS_SERVICE_KEY
-API_KEY: "thisisafakeapikey"
+API_KEY: "{{ lookup('password','/dev/null length=40') }}"
```

Comment:
Any idea how we could reuse the same value in the edxapp vault as they need to correspond? Cross-app secret generation...

Reply:
Maybe we should declare this improvement in an issue and implement it later?

Reply:
Maybe we can:

- decrypt the vault
- sed on the file to replace the line containing `COMMENTS_SERVICE_KEY:` with the good value
- encrypt it

But I agree with julien, in a new PR ?